source/documentation/introduction_to_the_vm_portal/topics/Graphical_User_Interface_elements.adoc

```diff
@@ -8,23 +8,39 @@ image::VM_screen.png[VM Portal screen]
 
 *Key graphical user interface elements*
 
-* image:1.png[title="Header bar"] `Header bar`
+* `Header bar`
 +
-The header bar contains the *Refresh* button, the btn:[User] drop-down button, and the btn:[Messages] drop-down button.
+The header bar contains the following buttons:
+
+* collapse/expand text labels in the side bar
+** btn:[Refresh] - refreshes the display
+** btn:[Account Settings]
```

Comment:
account settings -  enables setting of logged-in user preferences that will be persisted on server.